### ISSUE TITLE
Cannot run programs outside xbmc

### ISSUE THREAD

**Issue by wonx** [xbianonpi/xbian#136]
It's impossible to run a program outside xbmc, using plugins such as "Rom Collection Browser" or "Advanced Launcher". The program seems to start, but in the background (I can hear its sound), and the screen only shows xbmc.

I don't know how to attach a logfile here, but I also summited this bug here, where you can see it, in comment 2:

https://code.google.com/p/romcollectionbrowser/issues/detail?can=2&start=0&num=100&q=&colspec=ID%20Type%20Status%20Priority%20Milestone%20Owner%20Summary%20Modified&groupby=&sort=&id=136

Basically, I just want to be able to run the dillo web browser, and some emulated games.
